### DISCUSSION TITLE
i'm boggled RTL sats and bitrefill don't add up

### DISCUSSION THREAD

**Discussion by maximlomans** [Ride-The-Lightning/RTL#837] [General]
On RTL 350,000 sats is 180euros+/- , but 3,000,000 sats on Bitrefill is 60euros+/- 
what am i missing ?
the only i can think of is that Bitrefill is adding .00 for centimes ,where 3000000 comes down to 30,000.00 
... right ?
but then again on webanalysis.com 30,000sats comes to 15.92euro, i paid 60eu+/-,   so that doesn't add up either
i already tried reaching out to Bitrefill but to no avail, that is why i'm seeking enlightenment here 
: \

**Comment by saubyk** [comment]
Hi @marxenegls not sure how we can help you here, but the rate that you are observing on RTL is correct i.e. 350,000 sats is ~180 EUR (based on current rate).

Not clear, what you're trying to buy from Bitrefill. They only can help address your query.

**Reply by maximlomans** [reply]
you clearly haven't understood me - i bought 57eu+/- worth of btc in sats - for liquidity, they're pushing over 3000000sats which is 1,600eu+/-, so i wondered if i got the conversion wrong or RTL uses msats, but it doesn't ...
so i asked if you or anyone experienced a similar thing or understand what is actually happening 
 b.t.w congrats on the like, for not understanding me Suheb .. i think you have a new fan ; ) , besides me. i mean it , you were the only that actually replied to my queries , short but replied nonetheless : ) ++like

**Reply by saubyk** [reply]
Haven't bought channels from Bitrefill ever, so cannot comment on that specifically.

But I am sure you understand that buying sats and buying liquidity are different things. E.g. you can pay 57 EUR to Bitrefill and they open a channel of 3,000,000 sats *capacity* to you. The fee you paid is for the service of having a reliable channel opened to you, for a specified number of days, with particular channel capacity. It's not the same as buying sats from an exchange. These are different things.

What would be interesting to know is, on the channel which has been opened to your node, what's the local and remote balance?

I checked out the site and I can currently buy a 3,000,000 sat channel from Bitrefill for 23.32 EUR.
<img width="609" alt="Screen Shot 2021-10-19 at 7 02 12 PM" src="https://user-images.githubusercontent.com/39208279/138002226-dbb49709-73ec-4552-a9e7-ffeb2c46019b.png">

**Reply by maximlomans** [reply]
yes exactly what i thought initially either there's 30,000.00 or the liquidity and purchase are two different things, but i still don't get it, 
...then why is it telling that the 3,000000 sats is beyond my limit when trying to open a channel -with the amount that bitrefill itself tells me to enter ?

 i appreciate your input Sahbi, i mean Suheb ; )

**Comment by maximlomans** [comment]
i'm having difficulty accessing the advanced options on channels window

![rtl-channel_add](https://user-images.githubusercontent.com/6319093/138316598-9a4f04ab-bd63-4080-948e-70f3e39484e3.PNG)

**Reply by saubyk** [reply]
> i'm having difficulty accessing the advanced options on channels window
> 
> ![rtl-channel_add](https://user-images.githubusercontent.com/6319093/138316598-9a4f04ab-bd63-4080-948e-70f3e39484e3.PNG)

Not clear, what you're trying to do here.
From the screenshot I can see that you have 507,893 sats on your onchain balance and you're trying to open a channel for 3,000,000 sats. That's not how it works. You can only open the channels for the amount you have on your on-chain balance.

**Reply by maximlomans** [reply]
well, that's what they keep telling me i should enter for sats,
 it's what i said from the beginning, copied the sats they told me all 3,000000 sats copied! - i don't get either

![mlnsOfsats](https://user-images.githubusercontent.com/6319093/138493993-b54ba305-8555-478c-94f6-95c84d24d046.PNG)
 i've tried using LNurl, which i thought would solve it but it too fails to get the channel opened - channels was force closed remotely after 1day .. should i remove that invoice ?
i used phone qr reader , downloaded the file copied it into lnurl on thunderhub - it too failed

**Comment by maximlomans** [comment]
i think i got it!!!

i had zap setup, admin.macaroon and tls.cert - 
opened the bitrefill page acc. in brave browser, out of  sandboxie. 
opened lnurl app, it found zap, i had seen its call earlier but then it didn't have a conn to node yet..
it just said 'channel open' all green lit fonts flickered...

**Reply by saubyk** [reply]
So basically you were looking for lnurl capability. RTL doesn't support lnurl yet.

**Reply by saubyk** [reply]
They do have instructions on how do it via RTL...
<img width="587" alt="Screen Shot 2021-10-22 at 5 32 04 PM" src="https://user-images.githubusercontent.com/39208279/138525347-e386c4e9-74e1-4caf-9817-31efd25aff94.png">
